cpp/include/Ice/Object.h

```diff
@@ -46,40 +46,39 @@ namespace Ice
 
         /// Tests whether this object supports a specific Slice interface.
         /// @param s The type ID of the Slice interface to test against.
-        /// @param current The Current object for the invocation.
-        /// @return True if this object has the interface
-        /// specified by s or derives from the interface
-        /// specified by s.
+        /// @param current The Current object of the incoming request.
+        /// @return @c true if this object has the interface specified by @p s or derives from the interface specified
```

Comment:
Should we use markdown for `true` instead of @c? You started using Markdown in another PR would be more consistent to use it for all code if possible.

Reply:
Yes, we should.  Will fix in a follow-up PR.

The only downside of `word` vs `@c word` is the colorization with vscode-cpptools. It's minor.